tests/src/kinClient.intg.test.ts

```diff
@@ -131,13 +131,13 @@ describe("KinClient", async () => {
 				done();
 			}
 		});
-
 		let sendBuilder = await sender.buildSendKin({
 			address: receiver.publicAddress,
 			amount: 10,
 			fee: 100,
 			memoText: 'sending kin'
 		});
 		hash = await sender.submitTransaction(sendBuilder);
-	}, 120000);
+
+	}, 180000);
```

Comment:
why increasing so much the timeout?

Reply:
Sometimes it fails if the timeout is smaller.